tests/fixtures/fixture_ribametamod.py

```diff
@@ -17,7 +17,7 @@ def make_msw_model(
 
     idomain = gwf["GWF_1"]["dis"]["idomain"]
     if "layer" in idomain.dims:
-        idomain_layer1 = idomain.sel(layer=1)
+        idomain_layer1 = idomain.sel(layer=1, drop=True)
```

Comment:
why is this?

Reply:
I ran into an issue with some tests, where iMOD Python was complaining about faulty coordinates. It turned out as the layer coord was passed on to ``imod.util.spatial.get_cell_area``, which throws an error if there is any coord more than x, y, dx, and dy. This could be easily fixed by dropping the layer coordinate upon selecting the first layer in the fixture. It did not seem like the layer coord was necessary in the first place, so I think dropping it is a slight improvement regardless.